CAP/PackageInfo.g

```diff
@@ -102,6 +102,7 @@ Dependencies := rec(
   NeededOtherPackages := [ [ "AutoDoc", ">= 2016.02.16" ],
                            [ "ToolsForHomalg", ">= 2018.05.22" ],
                            [ "io", ">=0" ],
+                           [ "CAP", ">= 2019.01.01" ],
```

Comment:
```suggestion
                           [ "MonoidalCategories", ">= 2019.01.16" ],
```

This might fix the CI fail.

Reply:
CAP should not depend on MonoidalCategories anyway.

Reply:
In the recent commit it does not